Assets/HoloToolkit/Utilities/Scripts/BoundingBoxHelper.cs

```diff
@@ -0,0 +1,332 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// The BoundingBoxHelper class contains functions for getting geometric info from the non-axis-aligned 
+/// bounding box of a GameObject. These functions can be used to align another object to the center of
+/// a certain face or the center of an edge of a face... etc.
+/// The BoundingBoxHelper static function can be used for a one time calculation.
+/// The dynamic functions can be used to obtain boundingcube info on an object's Update loop. Operations
+/// are minimized in the dynamic use scenario.
+/// </summary>
+public class BoundingBoxHelper : MonoBehaviour
```

Comment:
Does this need to be a MonoBehaviour? Its [only use](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2212/files#diff-cca878975ff11b2420bc1b4d7e52b8baR249) is with the 'new' keyword, which causes a Unity warning.

![image](https://user-images.githubusercontent.com/3580640/40868278-31c99b90-65c0-11e8-8272-60972465ccdf.png)